README.rst

```diff
@@ -36,15 +36,17 @@ Example:
         username: deedee
         email: deedee@example.com
 
-    Profile:
+    - Profile:
       - user: dee
         name: Douglas
 
-    Group:
+    - Group:
       - name: Ramones
         members: [joey.profile, dee.profile]
 
--  The root of YAML contain ``mapper`` names i.e. ``User``
+-  The root of YAML contain ``mapper`` names e.g. ``User``, ``Profile`` etc
```

Comment:
oh, yes. i was referring to SQLAlchemy mappers... and I just wrote this a week ago :disappointed:

What about

The root of YAML contains a list of SQLAlchemy mapper names